include/aie/Dialect/AIE/IR/AIEAttrs.td

```diff
@@ -144,4 +144,15 @@ def BDDimLayoutArrayArrayAttr : ArrayOfAttr<
     /*eltName*/BDDimLayoutArrayAttr.cppClassName
 >;
 
+def CascadeDir: I32EnumAttr<"CascadeDir", "Directions for cascade",
+  [
+    I32EnumAttrCase<"South", 0>,
+    I32EnumAttrCase<"West", 1>,
+    I32EnumAttrCase<"North", 2>,
+    I32EnumAttrCase<"East", 3>
```

Comment:
Doesn't it make more sense to move the cases out of `WireBundle` and reuse them here [like this](https://github.com/llvm/llvm-project/blob/e6598b053dc71845423132ac0803b1ecd0b71d45/mlir/test/lib/Dialect/Test/TestEnumDefs.td#L18). This would 

1. Prevent renumbering/reassociating enum values (confusing)
2. Save from having to define  `CASCADE_DIR_TO_STRM_SW_PORT_TYPE` in CDODirect (would be able to reuse `WIRE_BUNDLE_TO_STRM_SW_PORT_TYPE`)

Reply:
I see what you mean, I'll do that! Thank you! :)